apps/competitions-api/src/routes/competitions/createCompetition.ts

```diff
@@ -0,0 +1,57 @@
+import { Router } from 'express';
+import { prisma } from '@competition-manager/prisma';
+import { z } from 'zod';
+import { parseRequest, AuthenticatedRequest, checkRole } from '@competition-manager/utils';
+import { Competition$, BaseCompetition$ } from '@competition-manager/schemas';
+
+export const router = Router();
+
+const Body$ = Competition$.pick({
+    name: true,
+    date: true,
+    method: true
+}).extend({
+    paymentPlanId: z.number(),
+    optionsId: z.array(z.number()).optional(),
+});
+
+router.post(
+    '/',
+    parseRequest('body', Body$),
+    checkRole('club'),
+    async (req: AuthenticatedRequest, res) => {
+        //stripe a faire
```

Comment:
Mettre TODO dans le comment pour pouvoir facilement faire une recherche
```suggestion
        // TODO: stripe
```